### ISSUE TITLE
[CAIA Intake] Transition Experience: V0 static "Get started" page

### ISSUE THREAD

**Issue by nkontrabecki** [department-of-veterans-affairs/va.gov-team#81963]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: `Transition Experience
- **OCTO product owner:** `Lisa Koenigsberg
- **Product name:** `Transition Experience
- **Product manager:**  Gene Dreyband
- **Slack channel:** [#ves-transition](https://dsva.slack.com/archives/C06C7RS27GD)
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
## About your initiative

Transitioning Servicemembers (TSMs) and Recently Separated Veterans (RSVs) are overwhelmed by the transition process. The transition process itself is a major risk for creating vulnerable populations. In addition to juggling many personal life challenges as they move into civilian life, TSMs and RSVs have no easy way to find the range of benefits and which they are eligible for.

### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [x] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`

The primary goal of the Veteran Transition Support program is to serve as a hub that can help the discovery of benefits, and ideally simplify the enrollment process for benefits with similar eligibility requirements. The goal is increasing the utilization of benefits, and increasing the proportion of Veterans that engage with VA post-service.

While the core value proposition is for TSMs and RSVs, the Veteran Transition Support project will also be useful for any Veteran re-engaging with VA, regardless of their time since service.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

Research review - complete
Conducted research - complete
Design Intent review - complete
Review feedback - complete

The team is currently updating the Figma mock ups for V0. We plan to develop and do limited user testing staging late May to early June 2024.

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [x]  Yes
- [ ]  No
 
 *Note: **If you check yes, we’ll reach out to discuss details about the content in the react widget. We use these widgets to display entry points for new products to a certain percentage of users who visit our static pages. Please refer to this [GitHub reference](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md) for dynamic content.
 
- [x]  I acknowledge that if I change the launch to be incremental, or change it from incremental to 100%, I must notify CAIA of this change as there is additional work involved that may impact deadlines.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `80976`[80976](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80976)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact <Product Outline>](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/vet-transition-support) `Product Outline`

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### Content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83835
```

**Comment by laurwill** [comment]
Hi @nkontrabecki ,

We connected with our OCTO leads Danielle and Mikki on this work, and it sounds like the V0 work for June 2024 will be a single resources and support informational page on VA.gov that will draw on existing content from the [Welcome Kit](https://www.va.gov/welcome-kit/). 

Let us know if you have any questions about that plan! cc @strelichl

**Comment by kristinoletmuskat** [comment]
@laurwill this is done right? This is that R+S page?

**Comment by laurwill** [comment]
Yep done, closing
